api/requirements.txt

```diff
@@ -0,0 +1,12 @@
+pip>=9
+setuptools>=26
+twine
+wheel>=0.29
+
+streamlit
+fastapi
+google-cloud-vision
+python-multipart
+python-dotenv
+requests
+uvicorn
```

Comment:
We will need to merge this file with the main requirements.txt . @rtnikki do you think it is better to wait until your part is done?

Reply:
We actually don't need to merge. API server should be fully independent of the main model and should just use pickle file to 'connect' to the model (same as we did for the taxi thing last week)

Reply:
Ok perfect!